src/main/java/org/apache/commons/imaging/formats/webp/chunks/WebPChunk.java

```diff
@@ -29,30 +30,50 @@
  * used by the parser.
  *
  * @see <a href="https://developers.google.com/speed/webp/docs/riff_container">WebP Container Specification</a>
- *
  * @since 1.0-alpha4
  */
 public abstract class WebPChunk extends BinaryFileParser {
     private final int type;
     private final int size;
     protected final byte[] bytes;
+    private int chunkSize;
```

Comment:
The `final` modifier should be added to it.

Reply:
Good catch! Done.